torch/_tensor.py

```diff
@@ -334,11 +334,12 @@ def __setstate__(self, state):
         # See Note [Don't serialize hooks]
         self.requires_grad, _, self._backward_hooks = state
 
-    def __repr__(self):
+    def __repr__(self, *, tensor_contents=None):
```

Comment:
huh, are you allowed to do this?

Reply:
I think it's generally discouraged, although it "works". This was done to avoid introducing a new function abstraction that subclass writers need to know about.